README.md

```diff
@@ -75,6 +75,18 @@ If you are editing source code, running the following command will allow hot-rel
 auspice develop --datasetDir data
 ```
 
+#### Environment variables
+
+The server looks for some environment variables. All are optional.
```

Comment:
Nitpick, but `/server/client/`. (It's been a perpetual source of confusion to people over the years as Auspice can mean both the server and the client.)

The server does also look for some variables, the most common being `PORT` and `HOST`, so over time we can include them.

And then there's variables which are essentially compiler arguments, e.g. `BABEL_*`, which I would categorise separately to the server.

Reply:
Thanks for the clarification. This is something I haven't looked into much and doesn't seem to be spelled out in docs. I'll look further and update docs in a future PR.